packages/devtools_app/lib/src/flutter/theme.dart

```diff
@@ -66,6 +66,9 @@ const devtoolsBlue = ColorSwatch<int>(600, {
   400: Color(0xFF13B9FD),
 });
 
+/// A short duration to use for animations.
+const shortDuration = Duration(milliseconds: 50);
```

Comment:
maybe give some guidance on when short duration should be used instead of the default duration for others adopting this theme.

Reply:
done